deploy/csv/annotations.json

```diff
@@ -0,0 +1,5 @@
+{
+    "categories": "OpenShift Optional",
+    "description": "Observatorium Operator provides the ability to install the components that comprise the Observatorium project.",
+    "repository": "https://github.com/observatorium/configuration"
```

Comment:
we will need to update these URLs since the repo is now named https://github.com/observatorium/deployments